### ISSUE TITLE
incompleted installation with MSYS2 installer.

### ISSUE THREAD

**Issue by strategist922** [msys2/msys2#104]
Hi Team,
    I use installer got from https://sourceforge.net/projects/msys2/files/latest/download. The file name is "msys2-x86_64-20170918.exe", I follow the guide in https://github.com/msys2/msys2/wiki/MSYS2-installation, but when finished with no error, I found the directory and subdirectory of  mingw32 and mingw64 are empty, there is no gcc toochain installed!?

   Did I miss somthing? I should download  mingw32 and mingw64 and install them?

**Comment by darkn3rd** [comment]
That is correct actually.  It's a minimalist install, you have to then install the toolchain:

```bash
pacman -S base-devel mingw-w64-x86_64-toolchain
````

**Comment by strategist922** [comment]
@darkn3rd , thanks, it seems work now :)

**Comment by strategist922** [comment]
https://www.americanexpress.com/taiwan/

**Comment by strategist922** [comment]
@darkn3rd, it seems also work if install 32 bit toolchain by using following commend, am I right?

pacman -S base-devel mingw-w64-i686-toolcha
